app/static/js/userProfile.js

```diff
@@ -239,7 +245,24 @@ $(document).ready(function(){
     * Background Check Functionality
     */
   // Updates the Background check of a volunteer in the database
+
+
+function enableLiveCustomValidityClearing() {
+  const allSelectors = [".passedBackgroundCheck"];
+  allSelectors.forEach(selector => {
+    $(selector).each(function () {
+      // Avoid rebinding listeners on already-bound elements
+      if (!$(this).data("has-clearing-listener")) {
+        $(this).on("input", function () {
+          this.setCustomValidity("");
+        });
+        $(this).data("has-clearing-listener", true); // flag it
+      }
+    });
+  });
+}
```

Comment:
We create this function twice on different pages. We should probably have it in base.js and you can just pass in the selectors you want to work with.